server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/UserRoutes.java

```diff
@@ -61,6 +67,44 @@
 import spark.Service;
 
 public class UserRoutes implements Routes {
+    interface UserCondition {
+        boolean satisfyCondition(UserResponse user);
+    }
```

Comment:
```suggestion
    interface UserCondition extends Predicate<Username> {

    }
```

Because this is what this is...